### PR TITLE
Kiali helm chart: add jaegerInCluserURL (1.5)

### PR DESCRIPTION
**Pull request by jotak**
Adding in-cluster-url to Kiali helm chart with default to "http://tracing.istio-system/jaeger"

Related to https://github.com/istio/istio.io/issues/6374
Similar to PR on master: https://github.com/istio/istio/pull/21255 , but more files are impacted

cc @jmazzitelli @gbaufake

### CODE REVIEW

**Comment by fpesce** [comment]
Tests failing with 
```
manifest-generate_test.go:187: unknown field "jaegerInClusterURL" in v1alpha1.KialiDashboardConfig
```

**Comment by jotak** [comment]
@fpesce yes indeed, it's the new field I'm adding. I guess I need to update files in `testdata` as well

**Comment by jotak** [comment]
Hi @fpesce , I'm trying to amend the code base wherever I find that my new helm chart variable needs to be added, but it seems I'm still missing something. Could you help me pointing what is missing?
Thanks.

**Comment by dgn** [comment]
@googlebot I consent.

File: install/kubernetes/helm/istio/charts/kiali/templates/configmap.yaml [hunk 1]
```diff
@@ -29,7 +29,7 @@ data:
     external_services:
       tracing:
         url: {{ .Values.dashboard.jaegerURL }}
-        in_cluster_url: {{ .Values.kiali.dashboard.jaegerInClusterURL }}
+        in_cluster_url: {{ .Values.dashboard.jaegerInClusterURL }}
```
Comment (jotak) [inline install/kubernetes/helm/istio/charts/kiali/templates/configmap.yaml]: oops good catch

**Comment by jotak** [comment]
thanks for the help @dgn !

**Comment by sdake** [comment]
@jotak nice work! Welcome to the Istio community. After inspection, it appears that the various PRs related to this issue have been rectified in master. It also appears the various backports have been done.

@istio/release-managers-1-5 Please ensure that PRs merge to master prior to accepting a backport into 1.5. The process should be PRs merge to master first. There are limited exceptions to this case that this PR didn't meet.

I am not sure why our cherrypick robot is not using the `-x` flag. @clarketm is there any opportunity to improve the cherrypick robot to pass this flag or help the release managers easily understand they are processing a backport (instead of an out of process direct merge to 1.5?). I am unfamiliar with the backport robot, although I am somewhat aware you have been digging into the bots codebase.

Cheers
-steve

**Pull request merged**